lib/core/src/display/symbol/material.rs

```diff
@@ -17,15 +17,17 @@ use crate::system::gpu::types::*;
 #[derive(Clone,Debug)]
 pub struct VarDecl {
     /// The GLSL type of the variable.
-    pub tp: glsl::PrimType,
+    pub tp : glsl::PrimType,
```

Comment:
Minor - we do not put spaces before `:` if it is not aligned with other lines.

Reply:
The coding style guide put the space in its example